vespalib/src/vespa/vespalib/data/slime/json_format.cpp

```diff
@@ -434,7 +434,7 @@ JsonDecoder::decodeNumber(Inserter &inserter)
         default:
             char *endp;
             int errorCode = insertNumber(inserter, isLong, value, &endp);
-            if (errorCode != 0) {
+            if (endp == value.c_str()) {
```

Comment:
this will allow overflow in addition to 'kind of underflow' and actual underflow, which might cause problems further down the road... (consider using common utility that allows underflow but not overflow...)

Reply:
done